### PR TITLE
Add Zulu 7

### PR DESCRIPTION
**Pull request by nathansgreen**
Azul's supported line of OpenJDK 7 builds.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
In [acceptable tasks](https://github.com/caskroom/homebrew-versions#acceptable-casks):

> Refuse legacy versions of gratis or open-source software, unless there is a clear demonstrable need for them.

**Comment by adidalal** [comment]
@vitorgalvao Should be the same as having old versions of Java, honestly (which we do allow). Some programs need them.

**Comment by vitorgalvao** [comment]
@adidalal Alright, will leave the `zulu` ones to you.

File: Casks/zulu7.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+cask 'zulu7' do
+  version '1.7.0_95,7.13.0.1'
+  sha256 '3744b75e621876934c3dc256491d5a3e58b73cd96e2b809c77a333f7e34debbe'
+
+  url "http://cdn.azul.com/zulu/bin/zulu#{version.after_comma}-jdk#{version[2..7].underscores_to_dots}-macosx_x64.dmg",
```
Comment (vitorgalvao) [inline Casks/zulu7.rb]: @nathansgreen Please change the interpolation here, though. See [the other `zulu`](https://github.com/caskroom/homebrew-cask/blob/cf459cdb18ed3d772d06b3976ddb045213981fd9/Casks/zulu.rb#L5).

**Comment by nathansgreen** [comment]
The Zulu build is actually newer than Oracle's builds and seems to be updated more frequently.

**Comment by nathansgreen** [comment]
@vitorgalvao, instead of adding a `caveats` section like Zulu, should I just use `zap`? I'm not sure if that's an appropriate use of zap.

**Comment by vitorgalvao** [comment]
`zap` is for configurations. `caveats` should be used.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 2293df997f18ae52053abcb6592ded6cb8a1c093. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**